specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
```

Comment:
This is formatted as an action, yet is a GET request. [Actions](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action) are always POST though.

Should this maybe be `POST /sip:checkNumberRouting` with payload
```json
{
   "targetPhoneNumber": "12345"
}
```
?

Reply:
@DominikMe @mbakalov @petrsvihlik @prprabhu-ms 
Based on the internal review we decided to make it a POST action `POST /sip:matchRoutesForNumber`
One more question from me is maybe we can keep `targetPhoneNumber` as a `query` parameter... It fits the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action) and I think is more abstract than specifying JSON body payload, and `targetPhoneNumber` is not data that is submitted to be stored/make any permanent change in the resource - so maybe no need to hide it in `body`. What do you think?

> :ballot_box_with_check: **YOU SHOULD** pattern your URL like this to perform an action on a collection
> **URL Pattern**
> ```text
> https://.../<resource-collection>:<action>?<input parameters>
> ```
> 
> **Example**
> ```text
> https://.../users:grant?access=read
> ```

Reply:
Alright, following what we agreed on in the review chat, will keep `targetPhoneNumber as `query` remembering to support bot `%2B` and `+` in the URI.